src/components/Wizard/CreateVmWizard/VmSettingsTab.js

```diff
@@ -32,20 +32,80 @@ import {
   USER_TEMPLATE_KEY,
   WORKLOAD_PROFILE_KEY,
 } from './constants';
-import { FormRow } from '../../Form/FormRow';
-import { isFieldDisabled, isFieldHidden, isFieldRequired } from './utils/vmSettingsTabUtils';
-import { getDefaultValue, getFieldHelp, getFieldId, getFieldTitle } from './initialState/vmSettingsTabInitialState';
+import { FormRowOPT } from '../../Form/FormRow';
+import { getVmSettings, isFieldDisabled, isFieldHidden, isFieldRequired } from './utils/vmSettingsTabUtils';
+import {
+  getDefaultValue,
+  getFieldHelp,
+  getFieldId,
+  getFieldTitle,
+} from './redux/initialState/vmSettingsTabInitialState';
 import { ImportProvider } from './providers/ImportProvider/ImportProvider';
-import { objectMerge } from '../../../utils';
+import { types, vmWizardActions } from './redux/actions';
+
+const Namespaces = ({ namespaces, ...props }) => {
```

Comment:
these 3 small components are just an optimization to map all the arrays on each keyboard stroke - e.g. description